docs/UserGuide.md

```diff
@@ -258,7 +258,7 @@ Undoes the previous command that the user has entered, which has changed the dat
 
 Format: `redo`
 
-### Undoing a command: `redo`
+### Redoing a command: `redo`
```

Comment:
I think it will be good to add in which specific commands can be redone (commands that change data within the JSON file).

Reply:
Agreed, but I think we can add this in a separate PR

Reply:
Okay if we're adding it in a separate PR, I think this is good to merge